green-ecolution/green-ecolution-backend
# Feature/link user to watering plan

Close #266 

TODO: 
- validate matching driving licence
- validate matching user role

**How to test:**
- Call the endpoints in swagger for watering plans and check if the GET methods (GetAll and GetByID) return UserIDs
- Check if a watering plan can be created with user IDs and not without user IDs
- Check if a watering plan can be updated with user IDs
